solidity/contracts/deposit/Deposit.sol

```diff
@@ -364,17 +333,72 @@ contract Deposit is DepositFactoryAuthority {
         );
     }
 
-    //
-    // FRAUD
-    //
+//---------------------------- LIQUIDATION FLOW ------------------------------//
+
+    /// @notice Notify the contract that the signers are undercollateralized.
+    /// @dev This call will revert if the signers are not in fact
+    ///      undercollateralized according to the price feed. After
+    ///      TBTCConstants.COURTESY_CALL_DURATION, courtesy call times out and
+    ///      regular abort liquidation occurs; see
+    ///      `notifyCourtesyTimedOut`.
+    function notifyCourtesyCall() public {
+        self.notifyCourtesyCall();
+    }
+
+    /// @notice Notify the contract that the signers' bond value has recovered
+    ///         enough to be considered sufficiently collateralized.
+    /// @dev This call will revert if collateral is still below the
+    ///      undercollateralized threshold according to the price feed.
+    function exitCourtesyCall() public {
+        self.exitCourtesyCall();
+    }
+
+    /// @notice Notifies the contract that the courtesy period has expired and
```

Comment:
nit: `Notifies` -> `Notify` for consistency

Reply:
:100::100::100: